tests/bats/test-bats-dockerimage.bats

```diff
@@ -2,7 +2,7 @@
 
 @test "With no cmd/args, the image return docker-compose version" {
 	result="$(docker run ${DOCKER_IMAGE_NAME})"
-	[ "$result" == "docker-compose 1.2.0" ]
+	[[ "$result" == *"docker-compose version: 1.3.0"* ]]
```

Comment:
Why using a double Square in that case ?
Also, why using the "star" pattern for the version ?